deps/v8/src/base/sys-info-zos.h

```diff
@@ -0,0 +1,54 @@
+// Copyright 2012 the V8 project authors. All rights reserved.
```

Comment:
Should we add this file to zoslib instead?

Reply:
are stuff in this file only use by v8? or it can also be used by other libraries?

Reply:
sys-info-zos,h itself is included only sys-info-zos.cc (as of v12..zos), but it does include stuff that should be useful for others when accessing system resources.